ote/src/cljs/ote/app/controller/email_notification_settings.cljs

```diff
@@ -0,0 +1,92 @@
+(ns ote.app.controller.email-notification-settings
+  "Controller for email notification settings"
+  (:require [tuck.core :as tuck :refer-macros [define-event]]
+            [ote.communication :as comm]
+            [ote.app.routes :as routes]
+            [ote.db.transit :as transit]
+            [ote.ui.form :as form]
+            [ote.localization :refer [tr]]
+            [ote.app.controller.common :refer [->ServerError]]))
+
+(declare ->LoadData ->RegionsResponse ->UserNotificationsResponse ->UpdateSettings)
+
+(defmethod routes/on-navigate-event :email-settings [_ app]
+  (->LoadData))
+
+(defn load-email-notifications-from-server! []
+  (comm/get! "pre-notices/regions"
+             {:on-success (tuck/send-async! ->RegionsResponse)
+              :on-failure (tuck/send-async! ->ServerError)}))
+
+(defn load-regions-from-server! []
+  (comm/get! "settings/email-notifications"
+             {:on-success (tuck/send-async! ->UserNotificationsResponse)
+              :on-failure (tuck/send-async! ->ServerError)}))
+
+(tuck/define-event LoadData []
+                   {:path [:email-settings]}
+                   (load-regions-from-server!)
+                   (load-email-notifications-from-server!)
+                   (-> app
+                       (assoc :regions-loading true)
+                       (assoc :user-notifications-loading true)))
+
+;; Region id's are strings in db. Change them to keywords
+(defn str->keyword [x]
```

Comment:
use strings in translation file and this can be removed

Reply:
yep. Changed to strings